apps/new/widget/page/projects/Editor.jsx

```diff
@@ -412,6 +440,8 @@ const ModalContainer = styled.div`
     padding-bottom: 0px !important;
     margin-bottom: -20px !important;
   }
+
+  font-family: "Poppins", sans-serif;
```

Comment:
How can we define font-family only in CSS

h1
h2
h3
p

....

Reply:
I asked Marie I believe, but it's not really possible.